### DISCUSSION TITLE
Piping babel output

### DISCUSSION THREAD

**Discussion by mbrahimi02** [babel/babel#12293] [General]
Is there a way to avoid having to build ES6 into a dist folder? I'm looking to add support for ES imports/exports in my express project and basically I was hoping it would be possible to pipe the compiled output straight into node/nodemon without generating any files on the filesystem.

**Comment by milesj** [comment]
@mbrahimi02 You can maybe use `babel-node`? Which is a stand-in for the `node` binary.

https://babeljs.io/docs/en/babel-node

**Comment by kaicataldo** [comment]
If you’re looking to access the transformed code, [`@babel/core`](https://babeljs.io/docs/en/babel-core)’s transformation APIs will return the generated output.

If you’re looking for a way to import a module and transform it for use in your application, [`@babel/register`](https://babeljs.io/docs/en/babel-register) or [`@babel/node`](https://babeljs.io/docs/en/babel-node) will allow you to run Babel on modules in your application.
